addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -81,12 +81,29 @@ export class ProductScreen extends ControlButtonsMixin(Component) {
         return this.currentOrder.orderlines?.reduce((items, line) => items + line.quantity, 0) ?? 0;
     }
     async updateSelectedOrderline({ buffer, key }) {
-        if (this.pos.numpadMode === "quantity" && this.pos.disallowLineQuantityChange()) {
-            const order = this.pos.get_order();
-            if (!order.orderlines.length) {
-                return;
+        const order = this.pos.get_order();
+        const selectedLine = order?.get_selected_orderline();
+        if (!selectedLine) {
+            return;
+        }
+        if (
+            this.pos.numpadMode === "quantity" &&
+            selectedLine?.isPartOfCombo() &&
+            !this.pos.disallowLineQuantityChange()
+        ) {
+            if (key === "Backspace") {
+                order.remove_orderline(selectedLine);
+            } else {
+                this.popup.add(ErrorPopup, {
+                    title: this.env._t("Invalid action"),
+                    body: this.env._t(
+                        "The quantity of a combo item cannot be changed. A combo can only be deleted."
+                    ),
+                });
             }
-            const selectedLine = order.get_selected_orderline();
+            return;
```

Comment:
Replace this behavior. Allow the user from changing the quantity. When the user tries to change the quantity, show the number popup to ask for the number. If 0, delete the line, otherwise, keep the line with the specified number.

Reply:
I think it's simpler to just click on the combo again, because the whole point of the combo is to select the combo lines, so most of the time you would not just modify the quantity, because you will probably want different combo lines in the new combo. 

And also if we allow changing the quantity we might run into unintuitive flows, for ex if i buy a combo consisting on a `burger` and a `soda` and then i want to add another `soda` i might be tempted to modify the quantity of the existing `soda`, but if we allow changing the quantity of combos, then changing the quantity of the existing `soda` will also change the quantity of the `burger`, which is not what i wanted to do.

Reply:
Okay. We keep the currently implemented behavior.